packages/@o3r/core/schematics/sdk/index.ts

```diff
@@ -0,0 +1,37 @@
+import { chain, externalSchematic, Rule, schematic, strings } from '@angular-devkit/schematics';
+import * as path from 'node:path';
+import { getPackageManager, getPackageRootFolder, getWorkspaceConfig } from '@o3r/schematics';
+import { NgGenerateSdkSchema } from './schema';
+import type { NgGenerateTypescriptSDKShellSchematicsSchema } from '@ama-sdk/schematics';
+import type { PackageJson } from 'type-fest';
+import { NgGenerateModuleSchema } from '../library/schema';
+
+/**
+ * Add an Otter compatible SDK to a monorepo
+ *
+ * @param options Schematic options
+ */
+export function generateSdk(options: NgGenerateSdkSchema): Rule {
+  const cleanName = strings.dasherize(options.name);
+
+  return (tree, context) => {
+    const defaultRoot = getPackageRootFolder(tree, 'library');
+    const scope = tree.exists('/package.json') && (tree.readJson('/package.json') as PackageJson).name?.split('/')?.[0]?.replace(/^@/, '') || 'sdk';
+
+    /** Path to the folder where generate the new SDK */
+    const targetPath = path.posix.resolve('/', options.path || defaultRoot, cleanName);
```

Comment:
I don't think this is working as expected on Windows
is that equivalent in this case :
```suggestion
    const targetPath = path.posix.join(options.path || defaultRoot, cleanName);
```
?